g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/source/dao/impl/LocalSourceDaoImpl.java

```diff
@@ -121,4 +122,12 @@ public boolean updateToBundle(ComponentMessagesDTO componentMessagesDTO) {
 		}
 	}
 
+
+	@Override
+	public List<RecordModel> getUpdateRecords(String productName, String version, long lastModifyTime) throws L10nAPIException{
+		// TODO Auto-generated method stub
+		LOGGER.error("bundle use the local disk media request the S3 API!");
+		throw new L10nAPIException("Local disk bundle can't support get update record method!");
```

Comment:
For local bundle situation, how about the the latest json file's modified time?

Reply:
For local bundle situation, we get update source record from sqllite, can't use latest json file's modified time

Reply:
should we implement it in this function? In case the build runs as local bundle and this feature still can be used.